src/usb_moded-config.c

```diff
@@ -1061,6 +1061,16 @@ char * config_get_android_manufacturer(void)
     {
         return ssu_name;
     }
+#else
+    /*Trying use /etc/hw-release to get MANUFACTURER_ID*/
+    GKeyFile *hwReleaseFile;
+    GError *error = NULL;
+
+    // Create a new GKeyFile object and a bitwise list of flags.
+    hwReleaseFile = g_key_file_new();
```

Comment:
'hwReleaseFile' is not released -> leak.

Reply:
handle = g_key_file_new() ; .... ; g_key_file_unref(handle);

There is no g_key_file_unref(hwReleaseFile) -> leak